### ISSUE TITLE
API Reference and Documentation links not working

### ISSUE THREAD

**Issue by frank6ix** [DroidPlanner/3DRServices#499]
I tried accessing the API Reference and API documentation links but none worked.
Documentation: http://android.dronekit.io/
API Reference: http://android.dronekit.io/javadoc/

**Comment by ZanzyTHEbar** [comment]
Indeed - it's very annoying

**Comment by hamishwillee** [comment]
@mrpollo Is this still supported?

**Comment by ZanzyTHEbar** [comment]
Yeah, it's still supported. The issue is just the hyperlinks and the website. According to their gitter chat the website has been down for awhile. The repo itself has a folder with a bunch of documentation though - which is the bareminimum to get started. I am using it currently for a project - it does work.

**Comment by mintheinwin** [comment]
Documentation links are still not working? Anyone can access?

**Comment by RossHacquebard** [comment]
Links are still broken for me also, does anyone have any other source for the API documentation?
